pkg/apis/serving/v1alpha2/explainer.go

```diff
@@ -23,8 +23,8 @@ import (
 type Explainer interface {
 	GetStorageUri() string
 	CreateExplainerContainer(modelName string, predictorHost string, config *ExplainersConfig) *v1.Container
-	ApplyDefaults()
-	Validate() error
+	ApplyExplainerDefaults(config *ExplainersConfig)
```

Comment:
Why change these interfaces? There's something nice about the Explainer/Predictor/Transformer sharing interfaces.

Reply:
We'd need to generalize the config object though. Perhaps a map is sufficient or a shared typed "KFServingConfig" object.

Reply:
ye the issue is with the config type, so have to change the name, maybe a shared type like you said.

Reply:
Fixed the interface by using EndpointsConfig